BuildToolsCliVersion.txt

```diff
@@ -0,0 +1 @@
+1.0.0-preview2-002733
```

Comment:
@sokket I just added 002733 to the list of builds available in this branch's install scripts. Please give it a shot.